### DISCUSSION TITLE
Sanity Integration is not working

### DISCUSSION THREAD

**Discussion by syedm1** [vercel/community#3558] [Bug Report]
### Page to Investigate

https://vercel.com/new/templates/next.js/sanity-visual-editing-demo

### Steps to Reproduce

-Try to deploy this  https://vercel.com/new/templates/next.js/sanity-visual-editing-demo
-Mandatory step to integrate sanity
-Integration is non-responsive when I click sign in with Google (I have an account in Sanity(
-Tested on Chrome and Mozilla (Windows 11)

**Comment by lucas-santosP** [comment]
I'm having the same issue here... I tried with different browsers and no change.

**Comment by amyegan** [comment]
Thanks for reporting this. It should be working now. Please let me know if you still trouble with it 🙏
